### PR TITLE
Translation #23

### PR DESCRIPTION
**Pull request by timozattol**
@CedricCook could you check the translations when you have time?

Would be awesome :smile:

### CODE REVIEW

File: resources/lang/en/ads.php [hunk 1]
```diff
@@ -1,85 +1,50 @@
...
 return [
 
 	'sections'     => [
-		'general'   => 'Information générale',
-		'details'   => 'Détails du job',
-		'main'      => "Corps de l'annonce",
-		'publisher' => 'Personne de contact',
+		'general'   => "General informations",
+		'details'   => "Job details",
+		'main'      => "Body of the ad",
```
Comment (zifeo) [inline resources/lang/en/ads.php]: Ad content?

File: resources/lang/en/ads.php [hunk 2]
```diff
@@ -1,85 +1,50 @@
...
-		'languages'          => 'Français, bon niveau anglais requis',
-
-		'contact_first_name' => 'Anne',
-		'contact_last_name'  => 'Onyme',
-		'contact_email'      => 'anne.onyme@epfl.ch',
-		'contact_phone'      => '+41 21 69 XX XX',
+		'title'              => "Pizza delivery, Neuroscience experiment",
+		'category_id'        => "",
+		'place'              => "St-Sulpice, remotely",
+		'description'        => "What kind of work is it? Are there any particular dispositions (schedule, team, prerequisite) ?",
```
Comment (zifeo) [inline resources/lang/en/ads.php]: `?` should not have any space in front in english

File: resources/lang/en/ads.php (hunk 1 above)
Comment (timozattol) [inline resources/lang/en/ads.php]: Maybe it's clearer yes! @CedricCook?

File: resources/lang/en/ads.php (hunk 2 above)
Comment (timozattol) [inline resources/lang/en/ads.php]: my bad, fixing this

File: resources/lang/en/ads.php (hunk 1 above)
Comment (timozattol) [inline resources/lang/en/ads.php]: "Ad content" better than "Body of the ad"?

**Comment by timozattol** [comment]
@CedricCook can you take a look? :smile:

**Pull request merged**